mmdeploy/backend/tensorrt/utils.py

```diff
@@ -13,15 +13,17 @@
 from .init_plugins import load_tensorrt_plugin
 
 
-def save(engine: trt.ICudaEngine, path: str) -> None:
+def save(engine: Any, path: str) -> None:
```

Comment:
Besides `trt.ICudaEngine` what else type might `engine` be?

Reply:
IHostMemory, which is the output of new API in TensorRT 8.5+

Reply:
https://docs.nvidia.com/deeplearning/tensorrt/api/python_api/infer/Core/Builder.html#tensorrt.Builder.build_serialized_network